gpii/node_modules/solutionsRegistry/src/solutions/com.freedomscientific/magic.json5

```diff
@@ -0,0 +1,1268 @@
+{
+  "name": "MAGic",
+  "key": "com.freedomscientific.magic",
+  "parents": ["gpii.solutionsRegistry.solution", "gpii.solutionsRegistry.context.windows"],
+  "capabilities": [
+    "http://registry\\.gpii\\.net/common/magnification/enabled"
+  ],
+  "launchHandlers": {
+    "launcher": {
+      "type": "gpii.launchHandlers.flexibleHandler",
+      "options": {
+        "verifySettings": true,
```

Comment:
The verifySettings and retryOptions arguments were removed from the codebase some years ago, all of these should be removed globally